.github/workflows/test.yml

```diff
@@ -62,18 +62,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # list whatever Terraform versions here you would like to support
         terraform:
-          - '1.0.*'
+          - '1.11.*'
```

Comment:
there are a lot of versions. so trying to cover ones that make sense - we can't cover all nor is it needed
we are not using (so far) anything special in the provider code. So it should be good enough.